libvast_test/vast/test/fixtures/filesystem.hpp

```diff
@@ -9,20 +9,20 @@
 #pragma once
 
 #include "vast/detail/pp.hpp"
+#include "vast/logger.hpp"
 
 #include <filesystem>
 
 namespace fixtures {
 
 struct filesystem {
-  filesystem() {
+  filesystem(const std::string& suite) : directory("vast-unit-test/" + suite) {
```

Comment:
```suggestion
  explicit filesystem(const std::filesystem::path& suite) : directory("vast-unit-test" / suite) {
```

It's a path, so let's just make that explicit in the signature. The same applies to other fixtures that inherit from this.

Reply:
Up to that point its just the stringified suite name, this variant seems more logical to me, what do you think?

```
explicit filesystem(const std::string& suite) : directory(std::filesystem::path{"vast-unit-test"} / suite) {
```

Reply:
`std::string_view` by value, then I'm happy! :)